sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -53,6 +54,9 @@ class EventHubProducerClient(ClientBase):   # pylint: disable=client-accepts-api
      or ~azure.core.credentials.AzureNamedKeyCredential
     :keyword bool buffered_mode: If True, the producer client will collect events in a buffer, efficiently batch,
      then publish. Default is False.
+    :keyword Union[ThreadPoolExecutor, int] buffer_concurrency: The ThreadPoolExecutor to be used for publishing events
+    or the number of workers for the ThreadPoolExecutor. Default is none and a ThreadPoolExecutor with the default number of workers
+    will be created https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor
```

Comment:
I think you need a proceeding space character on both lines 58 and 59 (i.e. multi-lines should be aligned to after the ":")